processor/src/chiplets/mod.rs

```diff
@@ -100,7 +100,7 @@ impl Chiplets {
     /// the execution trace at which the permutation started.
     pub fn permute(&mut self, state: HasherState) -> (Felt, HasherState) {
         let (addr, return_state, lookups) = self.hasher.permute(state);
-        self.bus.request_hasher_operation(lookups, self.clk);
+        self.bus.request_hasher_operation(&lookups, self.clk);
```

Comment:
With the new changes to remove the `lookups` from the `Hasher` struct, we should also provide the lookups to the chiplets bus in this method. (And we should follow a similar pattern in other places)